bin/l10n-sync.js

```diff
@@ -153,10 +152,10 @@ async function findUpstream() /*: Promise<string> */ {
     return upstream;
   } catch (error) {
     console.error(error);
-    throw new Error(stripIndent`
-      Couldn't find the upstream remote. Is it well configured?
-      We're looking for either devtools-html/perf.html or firefox-devtools/profiler.
-    `);
+    throw new Error(
+      "Couldn't find the upstream remote. Is it well configured?\n" +
+        "We're looking for either devtools-html/perf.html or firefox-devtools/profiler."
```

Comment:
BTW this makes me think that this would probably match `firefox-devtools/profiler-server` too, but it's not a big deal :-)

Reply:
Ah, that's right. That might be good to check. I guess it's unlikely that we will encounter profiler-server remote in our repo, but better safe than sorry :)